src/routes/docs/widget/quick-start/+page.md

```diff
@@ -193,15 +191,15 @@ Opening the modal will display the Widget in a "Lightbox" or modal dialog and ma
 
 The Widget requires a real DOM element on which to mount. Since the Inline component will be mounted within your application's controlled DOM, it's important to understand the lifecycle of how your framework mounts elements to the DOM.
 
-React, for example, uses the Virtual DOM, and the Inline component cannot mount to a Virtual DOM element. So, you will need to wait until the element has been property mounted to the real DOM before instantiating the Widget. For more information about this, please [see our tutorial using React with the Inline component](/docs/widget/tutorials#inline-with-react-and-webpack).
+React, for example, uses the Virtual DOM, and the Inline component cannot mount to a Virtual DOM element. So, you will need to wait until the element has been property mounted to the real DOM before instantiating the Widget.
```

Comment:
Would portals be something that someone would use to integrate the login-widget and react?
https://reactjs.org/docs/portals.html 

I think they may but admittedly, i've never used portals. Not sure if in this context it would be worth mentioning this?

Reply:
I don't know. I've heard of "portals" but have never used them. We'll likely need to do a bit of testing on that.